src/runtime_src/core/tools/common/reports/platform/ReportRyzenPlatform.cpp

```diff
@@ -62,6 +62,16 @@ ReportRyzenPlatform::writeReport(const xrt_core::device* /*_pDevice*/,
       }
     }
 
+    const boost::property_tree::ptree& tops = pt_platform.get_child("tops", empty_ptree);
+    if (!tops.empty()) {
+      _output << "\nTOPs\n";
```

Comment:
Should we say something like "Max Supported TOPs" or something?

Reply:
fixed.